src/api/overlays/components/OverlayComponent.vue

```diff
@@ -35,19 +35,27 @@
           class="c-overlay__contents js-notebook-snapshot-item-wrapper"
           tabindex="0"
         ></div>
-        <div v-if="buttons" class="c-overlay__button-bar">
-          <button
-            v-for="(button, index) in buttons"
-            ref="buttons"
-            :key="index"
-            class="c-button js-overlay__button"
-            tabindex="0"
-            :class="{ 'c-button--major': focusIndex === index }"
-            @focus="focusIndex = index"
-            @click="buttonClickHandler(button.callback)"
-          >
-            {{ button.label }}
-          </button>
+        <div v-if="buttons || showSuppressOption" class="c-overlay__footer">
+          <div class="c-overlay__suppress-option">
+            <div v-if="showSuppressOption">
+              <input v-model="suppress" type="checkbox" class="l-composite-control l-checkbox" />
+              <label>{{ suppressionText }}</label>
+            </div>
+          </div>
+          <div v-if="buttons" class="c-overlay__button-bar">
+            <button
+              v-for="(button, index) in buttons"
```

Comment:
`buttonIndex` to make the section explicit and readable

Reply:
Code updated. Thanks.